README.md

```diff
@@ -0,0 +1,26 @@
+ACA Required Temperature
```

Comment:
Should be "maximum" not "required".  The latter either implies the temperature must be exactly the required value, or if that is understand still leaves ambiguity about the directionality.